prov/shm/src/smr_av.c

```diff
@@ -70,6 +70,12 @@ static int smr_av_insert(struct fid_av *av_fid, const void *addr, size_t count,
 	util_av = container_of(av_fid, struct util_av, av_fid);
 	smr_av = container_of(util_av, struct smr_av, util_av);
 
+	if (smr_av->used + count > SMR_MAX_PEERS) {
+		FI_WARN(&smr_prov, FI_LOG_AV,
+			"AV insert failed. Expect inserting %zu AV entries, but only %zu available\n",
```

Comment:
nit: please limit line lengths to about 80 characters.  Yes, I know people don't like to break strings, but when there are things like %zu in the middle, searching on strings is useless anyway.

Reply:
Sure.